modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/servlet/taglib/util/ContentDashboardDropdownItemsProvider.java

```diff
@@ -95,7 +99,30 @@ public ContentDashboardDropdownItemsProvider(
 			() -> {
 				DropdownItem dropdownItem = new DropdownItem();
 
-				dropdownItem.putData("action", "showInfo");
+				dropdownItem.setData(
+					HashMapBuilder.<String, Object>put(
+						"action", "showInfo"
+					).put(
+						"className", contentDashboardItem.getClassName()
+					).put(
+						"classPK", contentDashboardItem.getClassPK()
+					).build());
+
+				ResourceURL resourceURL =
```

Comment:
Should probably move this declaration to just before https://github.com/liferay-frontend/liferay-portal/pull/152/files#diff-82a96a032d82cb080686b01630ae5968R124 or Brian will ask to declare `as used`.

Reply:
cc @cgoncas

Reply:
I might be wrong, but I think Brian prefers:

```java
ResourceURL resourceURL =
    _liferayPortletResponse.createResourceURL();

dropdownItem.setHref(String.valueOf(resourceURL));
```

instead of

```java
ResourceURL resourceURL =
    _liferayPortletResponse.createResourceURL();

// other stuff inn between

dropdownItem.setHref(String.valueOf(resourceURL));
```

Reply:
But the stuff in between is about setting params to the `resourceURL`, isn't it?

Reply:
Ah. Forgive my brain fart.

![tenor-174361035](https://user-images.githubusercontent.com/156388/88219933-afecfb80-cc38-11ea-8627-4d5af592264b.gif)

Reply:
🤣 🤣 🤣 🤣 🤣 🤣